tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_GaugeWave.cpp

```diff
@@ -25,38 +25,45 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 namespace {
-struct GaugeWaveProxy : gr::Solutions::GaugeWave {
-  using gr::Solutions::GaugeWave::GaugeWave;
+template <size_t Dim>
+struct GaugeWaveProxy : gr::Solutions::GaugeWave<Dim> {
+  using gr::Solutions::GaugeWave<Dim>::GaugeWave;
 
   template <typename DataType>
-  using variables_tags = gr::Solutions::GaugeWave::tags<DataType>;
+  using variables_tags =
+      typename gr::Solutions::GaugeWave<Dim>::template tags<DataType>;
 
   template <typename DataType>
   tuples::tagged_tuple_from_typelist<variables_tags<DataType>> test_variables(
-      const tnsr::I<DataType, 3>& x, const double t) const noexcept {
-    return variables(x, t, variables_tags<DataType>{});
+      const tnsr::I<DataType, Dim>& x, const double t) const noexcept {
+    return this->variables(x, t, variables_tags<DataType>{});
```

Comment:
Why `this->`?

Reply:
I don't actually know why it is needed, but clang gave an error and printed out
```
    return variables(x, t, variables_tags<DataType>{});
           ^
           this->
```
so I did that and the error went away.